launch/tier4_planning_launch/launch/scenario_planning/scenario_planning.launch.xml

```diff
@@ -23,6 +23,15 @@
     </include>
   </group>
 
+  <!-- obstacle velocity limiter -->
+  <group>
+    <set_remap from="~/input/trajectory" to="/planning/scenario_planning/scenario_selector/trajectory"/>
+    <set_remap from="~/output/trajectory" to="/planning/scenario_planning/obstacle_velocity_limiter/trajectory"/>
+    <include file="$(find-pkg-share obstacle_velocity_limiter)/launch/obstacle_velocity_limiter.launch.xml">
+      <arg name="vehicle_info_param_file" value="$(var vehicle_info_param_file)"/>
+    </include>
+  </group>
```

Comment:
[IMO] it is better to move to motion planning

Reply:
Done: 3413912d5